mpi-proxy-split/lower-half/Makefile

```diff
@@ -42,7 +42,7 @@ default: kernel-loader
 	${MPICXX} ${CFLAGS} -c -o $@ $<
 
 kernel-loader: get_symbol_offset.o copy_stack.o patch_trampoline.o lh_func_ptr.o mem_wrapper.o switch_context.o kernel_loader.o mmap-fixed-noreplace.o ${DMTCP_DEPS}
```

Comment:
This is fine.  Just a comment for your own information.  Normally, gcc (or clang) will automatically link with libatomic.so.  But for some MPICXX it's hard to know if they cover it.  So, this is good as a safety measure.

Reply:
It's automatically linked on Perlmutter. However, on Discovery, the compiler complains about undefined functions in the library. So I add `-latomic` here for safety.